lib/foreman_rh_cloud.rb

```diff
@@ -3,17 +3,27 @@
 require 'uri'
 
 module ForemanRhCloud
+  def self.on_premise_url
+    return unless ForemanRhCloud.with_insights_on_premise?
+    port = ENV['INSIGHTS_ADVISOR_PORT'] || "8000"
+    "http://#{ForemanRhCloud.foreman_host.fqdn || 'localhost'}:#{port}"
```

Comment:
```suggestion
    "https://#{ForemanRhCloud.foreman_host.fqdn || 'localhost'}:#{port}"
```

Reply:
(or make it configurable… but we need SSL for real deployments)

Reply:
@parthaa :eyes:

Reply:
I could add another env variable for the protocol?

Reply:
updated :+1: